docs/samples/Microsoft.ML.Samples/Dynamic/NgramExtraction.cs

```diff
@@ -27,20 +28,29 @@ public static void Example()
             // A pipeline to tokenize text as characters and then combine them together into n-grams
             // The pipeline uses the default settings to featurize.
 
-            var charsPipeline = ml.Transforms.Text.TokenizeIntoCharactersAsKeys("Chars", "SentimentText", useMarkerCharacters: false);
-            var ngramOnePipeline = ml.Transforms.Text.ProduceNgrams("CharsUnigrams", "Chars", ngramLength: 1);
-            var ngramTwpPipeline = ml.Transforms.Text.ProduceNgrams("CharsTwograms", "Chars");
+            var charsPipeline =
+                ml.Transforms.Text.TokenizeIntoCharactersAsKeys("Chars",
+                    "SentimentText", useMarkerCharacters: false);
+            var ngramOnePipeline =
+                ml.Transforms.Text.ProduceNgrams("CharsUnigrams", "Chars",
+                    ngramLength: 1);
+            var ngramTwpPipeline =
+                ml.Transforms.Text.ProduceNgrams("CharsTwograms", "Chars");
             var oneCharsPipeline = charsPipeline.Append(ngramOnePipeline);
             var twoCharsPipeline = charsPipeline.Append(ngramTwpPipeline);
 
             // The transformed data for pipelines.
-            var transformedData_onechars = oneCharsPipeline.Fit(trainData).Transform(trainData);
-            var transformedData_twochars = twoCharsPipeline.Fit(trainData).Transform(trainData);
+            var transformedData_onechars =
+                oneCharsPipeline.Fit(trainData).Transform(trainData);
+            var transformedData_twochars =
+                twoCharsPipeline.Fit(trainData).Transform(trainData);
 
             // Small helper to print the text inside the columns, in the console. 
-            Action<string, IEnumerable<VBuffer<float>>, VBuffer<ReadOnlyMemory<char>>> printHelper = (columnName, column, names) =>
+            Action<string, IEnumerable<VBuffer<float>>, VBuffer<ReadOnlyMemory<char>>
+            > printHelper = (columnName, column, names) =>
```

Comment:
>            > [](start = 0, length = 13)

let manually fix any line that starts with >

this looks like lambda with errors.